### ISSUE TITLE
Please add stop and restart 

### ISSUE THREAD

**Issue by coder-efay** [jayasme/SpotlightLyrics#1]
Hi, thanks for sharing this project. Here is just a small problem
The stop button on your Demo is just a pause, if I click start, it will resume.
How do I stop the whole lyrics and restart from the beginning?
Thank you very much

**Comment by coder-efay** [comment]
The following is not working, it just started at where it paused 
```
// Start it over
lyricsView.timer.seek(toTime: 0)
lyricsView.timer.play()
```
Please advice how to start over from the beginning

**Comment by jayasme** [comment]
Thanks for submitting the issue, I’ll check the code and make a solution ASAP, apologies for my lated reply.

**Comment by jayasme** [comment]
@coder-efay  Hi, I've released the newest version 0.1.5 for fixing this issue, please do the following to update the dependency:
```
pod update
pod install
```
thank you for your patience waiting and apologies again for my late reaction.
